src/types/packagejson.ts

```diff
@@ -50,7 +50,7 @@ export interface PackageJson {
     /**
      * You should specify a license for your package so that people know how they are permitted to use it, and any restrictions you’re placing on it.
```

Comment:
```suggestion
     * You should specify a license for your package so that people know how they are permitted to use it, and any restrictions you’re placing on it.
```